api/mfa.go

```diff
@@ -2,22 +2,25 @@ package api
 
 import (
 	"bytes"
-	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"github.com/aaronarduino/goqrsvg"
+	"github.com/ajstarks/svgo"
+	"github.com/boombuler/barcode/qr"
 	"github.com/netlify/gotrue/crypto"
 	"github.com/netlify/gotrue/models"
 	"github.com/netlify/gotrue/storage"
 	"github.com/pquerna/otp/totp"
-	"image/png"
 	"net/http"
+	"strconv"
 	"time"
 )
 
 type EnrollFactorParams struct {
 	FriendlyName string `json:"friendly_name"`
 	FactorType   string `json:"factor_type"`
 	Issuer       string `json:"issuer"`
+	QRCodeSize   string `json:"qr_code_size"`
```

Comment:
hmm why do we need to take in the size of the qr code as a param if it's gonna be a SVG

Reply:
Oh included for configurability -- the SVG code generator has a [block size](https://github.com/aaronarduino/goqrsvg/blob/master/goqrsvg.go#L29) which is supposed to correspond to the size of a pixel block

We can set a reasonable default (say 3) if that's better since the SVG can be resized